### PR TITLE
Added more Unit Tests

### PR DESCRIPTION
**Pull request by diegopacheco**
@ipapapa 

- Added more Unit Tests
- Removed imports which was not use - code clean up.

Cheers,
Diego Pacheco

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,12 +41,14 @@ subprojects {
...
     dependencies {
+		
         compile "org.yaml:snakeyaml:latest.release"
+        
         compile "com.google.inject:guice:3.0"
         compile "com.google.inject.extensions:guice-servlet:3.0"
-        compile "com.sun.jersey.contribs:jersey-guice:1.9.1"
-        compile "com.sun.jersey:jersey-core:1.17.1"
-        compile "com.sun.jersey:jersey-server:1.17.1"
+        compile ("com.sun.jersey.contribs:jersey-guice:1.9.1")
```
Comment (ipapapa) [inline build.gradle]: Is this change needed? It seems we are diverting from the standard formatting of the rest.

File: dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java [hunk 2]
```diff
@@ -15,6 +15,13 @@
  */
 package com.netflix.dynomitemanager.supplier;
 
+import java.util.ArrayList;
```
Comment (ipapapa) [inline dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java]: why do we need to change these import to a different place?

File: build.gradle (hunk 1 above)
Comment (diegopacheco) [inline build.gradle]: @ipapapa  actually there is no changes. I was excluding a library thats why you see the the () IF you want I can revert-it.

File: dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java (hunk 2 above)
Comment (diegopacheco) [inline dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java]: no reason. I use eclipse and run CTRL + SHIFT + O to organized imports... I have revert-it as you like it...

File: build.gradle (hunk 1 above)
Comment (diegopacheco) [inline build.gradle]: DONE @ipapapa

File: dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java (hunk 2 above)
Comment (ipapapa) [inline dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java]: that is fine with me to reorganize them, just asking the reason :)
